src/main/webapp/individualFileTable.xhtml

```diff
@@ -0,0 +1,60 @@
+<ui:composition
+    xmlns="http://www.w3.org/1999/xhtml"
+    xmlns:h="http://java.sun.com/jsf/html"
+    xmlns:f="http://java.sun.com/jsf/core"
+    xmlns:ui="http://java.sun.com/jsf/facelets"
+    xmlns:p="http://primefaces.org/ui"
+    xmlns:c="http://xmlns.jcp.org/jsp/jstl/core"
+    xmlns:jsf="http://xmlns.jcp.org/jsf"
+    xmlns:pt="http://java.sun.com/jsf/passthrough"
+    xmlns:cc="http://java.sun.com/jsf/composite"
+    xmlns:o="http://omnifaces.org/ui"
+    xmlns:iqbs="http://xmlns.jcp.org/jsf/composite/iqbs">
+    <p:outputPanel>
+        <div id="fileInfoInclude-filesTable" class="col-sm-12 row">
+            <div class="pull-left col-file-thumb">
```

Comment:
I just wanted to note that @mheppler made a comment above about file name conventions for xhtml files. From my perspective, we are not at all consistent. My approach would probably be to start by asserting a preference in the dev guide on the "Coding Style" page. Then we could work toward renaming existing files to comply with the naming standard.